libs/estdlib/src/gen_server.erl

```diff
@@ -326,8 +335,20 @@ call(Name, Request, TimeoutMs) when is_atom(Name) ->
             call(Pid, Request, TimeoutMs)
     end;
 call(Pid, Request, TimeoutMs) when is_pid(Pid) ->
-    Ref = erlang:make_ref(),
-    call_internal(Pid, {'$call', {self(), Ref}, Request}, TimeoutMs).
+    MonitorRef = monitor(process, Pid),
+    Pid ! {'$call', {self(), MonitorRef}, Request},
+    receive
+        {'DOWN', MonitorRef, process, Pid, normal} ->
+            {error, noproc};
+        {'DOWN', MonitorRef, process, Pid, Reason} ->
+            erlang:exit(Reason);
```

Comment:
Reading OTP documentation, I find that the exit term is not `Reason` but `{Reason, Location}`.

https://www.erlang.org/doc/man/gen_server.html#call-2

I wonder what can be tested before #713 is fixed, though.

Reply:
Fixed (to the extent possible)